Author: Dann Frazier <dannf@debian.org>
Description: When _XOPEN_SOURCE is defined, the subsequent #include <unistd.h>
will define a correct function prototype for the crypt function.  This avoids
segfaults on architectures where the size of a pointer is greater than the size
of an integer (ia64 and amd64 are examples).  This may be detected by looking
for build log lines such as the following:
auth_shadow.c:183: warning: implicit declaration of function ‘crypt’
auth_shadow.c:183: warning: cast to pointer from integer of different size
--- trunk.orig/saslauthd/auth_shadow.c
+++ trunk/saslauthd/auth_shadow.c
@@ -1,3 +1,4 @@
+#define _XOPEN_SOURCE
 #define PWBUFSZ 256 /***SWB***/
 
 /* MODULE: auth_shadow */
